documentation/jetty-documentation/src/main/asciidoc/operations-guide/jaspi/chapter.adoc

```diff
@@ -61,8 +63,8 @@ The `CallerPrincipalCallback` and `GroupPrincipalCallback` do not require use of
 
 Jetty provides an implementation of the `AuthConfigFactory` interface which is used to register `AuthConfigProviders`. This can be replaced by a custom implementation by adding a custom module which provides `auth-config-factory`.
 This custom module must reference an XML file which sets a new instance of the `AuthConfigFactory` with the static method `AuthConfigFactory.setFactory()`.
-For an example of this see the `jaspi-default-auth-config-factory` module, which provides the default implementation used by Jetty.
+For an example of this see the `ee{9,10}-jaspi-default-auth-config-factory` module, which provides the default implementation used by Jetty.
 
 ----
-include::{JETTY_HOME}/modules/jaspi-default-auth-config-factory.mod[]
+include::{JETTY_HOME}/modules/ee9-jaspi-default-auth-config-factory.mod[]
```

Comment:
ee9 -> ee10

Reply:
@gregpoulos still using ee9 here? There is no such module for ee10?

Reply:
Yikes, apparently I missed this one. It should be updated – I'll post a fix.